recipes/mypy-boto3-kinesis/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "mypy-boto3-kinesis" %}
+{% set version = "1.36.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
+  sha256: fb469950ec1e38f77570c14591ae8fcd6b65fbca0a48c0ba24458cd9e585bb5e
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python {{ python_min }}
+    - setuptools
+  run:
+    - boto3
+    - python >={{ python_min }}
+    - typing_extensions
+
+test:
+  imports:
+    - boto3
+    - mypy_boto3_kinesis
+  commands:
+    - pip check
+  requires:
+    - pip
+    - python {{ python_min }}
+
+about:
+  home: https://github.com/youtype/mypy_boto3_builder
+  summary: Type annotations for boto3 Kinesis {{ version }}service generated with mypy-boto3-builder
+  dev_url: https://github.com/youtype/mypy_boto3_builder
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - stevenschaerer
```

Happy to help with maintaining the recipe.

```suggestion
    - stevenschaerer
    - janjagusch
```